docs/tutorial/testing-on-headless-ci.md

```diff
@@ -47,14 +48,15 @@ install:
 
 ### Jenkins
 
-For Jenkins, a [Xvfb plugin is available](https://wiki.jenkins-ci.org/display/JENKINS/Xvfb+Plugin).
+For Jenkins, an [Xvfb plugin is available](https://plugins.jenkins.io/xvfb/).
 
 ### Circle CI
 
-Circle CI is awesome and has Xvfb and `$DISPLAY`
-[already set up, so no further configuration is required](https://circleci.com/docs/environment#browsers).
+Circle CI has Xvfb and the `$DISPLAY` variable
+[already set up](https://circleci.com/docs/2.0/browser-testing/), so no
```

Comment:
This link doesn't explicitly say that xvfb is setup and this link seems to indicate only certain legacy images have xvfb installed: https://circleci.com/docs/2.0/circleci-images/#pre-installed-tools

Reply:
Very good catch. Sorry for the careless addition -- I'll try to do some more research on this.